### PR TITLE
Restructure processing logic

### PR DESCRIPTION
**Pull request by darkwing**
This is a simple update, more will come, so don't merge yet.

### CODE REVIEW

File: hook.js [hunk 1]
```diff
@@ -57,33 +57,18 @@ function processPullRequest(destinationRepo, originRepo, originBranch, prNumber,
...
-                process(githubClient, originRepo, originBranch, file, config, handleIncompatibility);
+                });
+
             });
+
         });
-    }).catch(function(error) {
-        logger.error(error);
-    });
+    }).catch(logger.error);
```
Comment (openjck) [inline hook.js]: Clever! :+1:

**Comment by openjck** [comment]
Needs a rebase

**Comment by openjck** [comment]
AFAICT, the rebase is only needed because the `filename` variable was added. So instead of doing a three-way merge, another option would be to copy these files out, create a new branch off of master, copy the files back in, and manually re-add the `filename` variable. Just a different way of getting to the same goal.

File: processor.js [hunk 2]
```diff
@@ -8,6 +10,30 @@ var stylus = require('stylus');
...
 
+
+
+/**
+ * Router for processing files CSS and Stylus
+ */
+function process(githubClient, repo, branch, file, config, handleIncompatibility) {
+    var processor;
+
+    // Test and report on this file if it's a stylesheet
```
Comment (openjck) [inline processor.js]: We can probably remove this comment now. The function comment says about the same thing.

**Comment by openjck** [comment]
Looks good. Might be nice to run a `gulp beautify` just for code style consistency.

**Comment by darkwing** [comment]
Updated!

**Pull request merged**